src/Symfony/Component/Workflow/Tests/DefinitionTest.php

```diff
@@ -18,7 +18,7 @@
 
 class DefinitionTest extends TestCase
 {
-    public function testAddPlaces()
+    public function testAddPlaces(): void
```

Comment:
We don't use the `void` return type in our tests IIRC

Reply:
Correct

Reply:
Oups sorry about that @guillaumeVDP